pkg/apis/resources/v1alpha1/types.go

```diff
@@ -169,6 +169,9 @@ type ManagedResourceStatus struct {
 	// Resources is a list of objects that have been created.
 	// +optional
 	Resources []ObjectReference `json:"resources,omitempty"`
+	// SecretChecksum is the checksum of referenced secret's data.
+	// +optional
+	SecretChecksum string `json:"secretChecksum,omitempty"`
```

Comment:
Nit: make it clear that it's about all secrets.
```suggestion
	// SecretsChecksum is the checksum of referenced secrets' data.
	// +optional
	SecretsChecksum string `json:"secretsChecksum,omitempty"`
```